flowtools/txt_diagnosis/txtdiagnosis.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+######################################################################
+#                  Copyright (c) 2016 Northrop Grumman.
+#                          All rights reserved.
+######################################################################
+import pandas as pd
+from argparse import ArgumentParser
+import sys
+
+
+def is_number(s):
+    try:
+        float(s)
+        return True
+    except ValueError:
+        return False
+
+
+def error_report(input_file, fname, output_file):
+    errors = 0
+    df = pd.read_table(input_file)
+    with open(output_file, "w") as outf:
+        for cols in df.columns.values:
+            if df[cols].count() != len(df[cols]):
```

Comment:
Please give it a go at using the pandas dataframe more natively here for finding out in a couple of lines the offending lines with NaNs, and then reporting that in the same way.